README.md

```diff
@@ -81,6 +81,37 @@ thumbnail generation.
 $ bundle exec sidekiq
 ```
 
+## Update instructions
+
+### 1. Update the code
+```bash
+$ git pull
+```
+
+### 2. Update and install required dependencies
+```bash
+$ bundle install
+```
+
+### 3. Run the database migrations 
+```bash
+$ bundle exec rake db:migrate RAILS_ENV=development
```

Comment:
`development` is the default so it's fine to just run `bundle exec rake db:migrate`